### PR TITLE
Interactive tutorial todo list

### PR DESCRIPTION
**Pull request by runrevelanor**

### CODE REVIEW

**Comment by livecodeali** [comment]
@livecode-vulcan review ok abd9401

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok abd9401092d2ceea6ce96bf90e7756a77f13a47b

File: Toolset/palettes/tutorial/courses/First Run/tutorials/To Do List/lessons/App Lesson.txt [hunk 1]
```diff
@@ -488,7 +481,7 @@ step "Test Delete Task"
 	The task will be deleted from the list.
 action
 	highlight widget "delete"
-	wait until the text of field "taskList" is ""
+	wait until the text of field "taskList" is changed
```
Comment (livecodeali) [inline Toolset/palettes/tutorial/courses/First Run/tutorials/To Do List/lessons/App Lesson.txt]: This needs a `with default ""`

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 5af1d14

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 5af1d14daf9ae3391e3889a40fa8a54783f565b7

**Pull request merged**